terraform/schema_14110.yaml

```diff
@@ -944,17 +955,79 @@ variables:
                     - ${subnet_type}
                     - "Use Public Subnet"
     type: boolean
-    default: true
+    default: false
     title: "Provision Bastion Instance"
     description: "Provision a bastion compute instance on a public subnet to provide access to the WebLogic Server compute instances on a private subnet. If not selected, you must check the status of the compute instances after the stack job is complete."
 
+  is_rms_private_endpoint_required:
+    visible:
+      and: 
+        - ${orm_create_mode}
+        - and:
+            - not:
+                - ${create_new_subnets}
+            - not:
+                - ${create_new_vcn}
+            - not:
+                - eq: 
+                    - ${subnet_type}
+                    - "Use Public Subnet"
+    type: boolean
+    default: true
+    title: "Provision Resource Manager Private Endpoint"
+    description: "Provision a bastion compute instance on a public subnet to provide access to the WebLogic Server compute instances on a private subnet. If not selected, you must check the status of the compute instances after the stack job is complete."
```

Comment:
Why is this mentioning creating a bastion compute instance?

Reply:
This needs to be changed. I have noted this down to talk with Vanaja for better wording.